recOrder/acq/acquisition_workers.py

```diff
@@ -942,39 +938,30 @@ def _save_imgs(self, birefringence, phase, meta=None):
         """
         prefix = self.calib_window.save_name
 
-        if birefringence is not None:
+        name = (
+            f"ReconstructionSnap.zarr"
+            if not prefix
+            else f"{prefix}_ReconstructionSnap.zarr"
+        )
 
-            name = (
-                f"BirefringenceSnap.zarr"
-                if not prefix
-                else f"{prefix}_BirefringenceSnap.zarr"
-            )
-            with open_ome_zarr(
-                os.path.join(self.snap_dir, name),
-                layout="fov",
-                mode="w-",
-                channel_names=["Retardance", "Orientation", "BF", "Pol"],
-            ) as dataset:
-                if birefringence.ndim == 3:
-                    birefringence = birefringence[:, np.newaxis, ...]
-                dataset["0"] = birefringence[np.newaxis, ...]
-                dataset.zattrs["recOrder"] = meta
+        dataset = open_ome_zarr(
+            os.path.join(self.snap_dir, name),
+            layout="fov",
+            mode="w-",
+            channel_names=["Retardance", "Orientation", "BF", "Pol"],
+        )
+        if birefringence.ndim == 3:
+            birefringence = birefringence[:, np.newaxis, ...]  # CYX -> CZYX
+        birefringence = birefringence[np.newaxis, ...]  # CZYX -> TCZYX
+        dataset["0"] = birefringence
 
         if phase is not None:
-            name = (
-                f"PhaseSnap.zarr" if not prefix else f"{prefix}_PhaseSnap.zarr"
-            )
+            dataset.append_channel("Phase", resize_arrays=True)
```

Comment:
We used to have `Phase2D` or `Phase3D` channel names depending on the reconstruction algorithm. Do we want to change that behavior?

Reply:
Nice catch! Thank you.